schemas/string_interpolation/transforms/option_label_from_value.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/option_label_from_value.json",
+  "option_label_from_value": {
+    "description": "Get the label of an answer option based on its value. Takes two arguments, the option value and the answer id",
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["option_label_from_value"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "value": {
+            "$ref": "https://eq.ons.gov.uk/string_interpolation/definitions.json#/placeholder_value_sources"
+          },
+          "answer_id": {
+            "type": "string"
+          }
```

Comment:
This answer id should be validated in code to ensure it exists. It is done for rules code, but since this is an arbitrary string we want to ensure it is valid at build time. https://github.com/ONSdigital/eq-questionnaire-validator/blob/b0a937b566b6a172d82159ce0c0366bc57bb56b1/app/validators/rules/rule_validator.py#L187-L195

Reply:
We should also validate that this answer ID also has an answer type which allows answers options.